python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -425,8 +425,8 @@ def check_run_worker_health(self, run: PipelineRun):
                     container_str = "Container"
                 return CheckRunHealthResult(
                     WorkerStatus.FAILED,
-                    f"ECS task failed. Stop code: {t.get('stopCode')}. Stop reason {t.get('stopReason')}. "
-                    f"{container_str} {[c.get('name') for c in failed_containers]} failed."
+                    f"ECS task failed. Stop code: {t.get('stopCode')}. Stop reason: {t.get('stoppedReason')}. "
```

Comment:
verified that this is the correct key as per boto docs

Reply:
oops